steal.lic

```diff
@@ -16,13 +16,21 @@ class Steal
 
     @stealing_options = get_data('stealing').stealing_options.reject { |target| settings.dont_steal_list.include?(target['id'].to_i) }
     @stealing_bag = settings.stealing_bag
-    hometown_data = get_data('town')[settings.hometown]
-    @bin_id = hometown_data['thief_bin'] && hometown_data['thief_bin']['id']
+    stealing_hometown = settings.hometown 
+    stealing_hometown = settings.force_bin_town if settings.force_bin_town
     @stealing_towns = settings.stealing_towns
     @bin_stolen = settings.bin_stolen
-    if @bin_stolen && !@bin_id
-      DRC.message("*** Binning is not yet supported in #{settings.hometown}")
+    if DRStats.thief?
+      @bin_id = DRCT.get_hometown_target_id(stealing_hometown,'thief_bin')
+      if !(@bin_id)
+        DRC.message("Binning not supported in #{stealing_hometown}.  You should turn it off or change hometown/force_bin_town.")
+        @bin_stolen = false
+        pause 5
+      end
+    else
+      DRC.message("You are not a thief.  You can't use thief bins.  You should turn it off.")
```

Comment:
Looks like in this update that the check for if `@bin_stolen` is set was removed from original line 23. I think this message will appear every time for non-thieves. It doesn't hurt, but may give false positives.